src/System.Net.Primitives/src/System/Net/CookieContainer.cs

```diff
@@ -520,7 +520,7 @@ private int ExpireCollection(CookieCollection cc)
             }
         }
 
-        internal void Add(CookieCollection cookies)
+        public void Add(CookieCollection cookies)
```

Comment:
I have added a test for Add method , even though the new Add method does little and the other Add method which does the work is already covered.